twindle-cli/twitter/scraping/index.js

```diff
@@ -1 +1,90 @@
-// once puppeteer is ready we can bring it into this folder
+const puppeteer = require("puppeteer");
+const { waitFor } = require("../utils/helpers");
+
+/**
+ * Get tweets(even older than 7 days) using puppeteer
+ * @param {string} tweetID
+ */
+const getTweetIDs = async (tweetID) => {
+  const pageURL = `https://twitter.com/anyone/status/${tweetID}`;
+
+  // Modify this variable to control the size of viewport
+  const factor = 0.2;
+  const height = Math.floor(2000 / factor);
+  const width = Math.floor(1700 / factor);
+
+  const browser = await puppeteer.launch({
+    headless: true,
+    defaultViewport: {
+      width,
+      height,
+    },
+    args: [`--force-device-scale-factor=${factor}`, `--window-size=${width},${height}`],
+  });
+
+  const context = browser.defaultBrowserContext();
+  context.clearPermissionOverrides();
+  context.overridePermissions("https://twitter.com", ["clipboard-read", "clipboard-write"]);
```

Comment:
@Mira-Alf please check the main branch again. The scrapping logic part improved by @PuruVJ 

We no need to use clipboard read write which is not very clean.

Reply:
Yes @PuruVJ please merge in your changes to this branch.

Reply:
Will do boss! 😁

Reply:
Actually they're already merged! I forgot to remove permissions override. Doesn't matter much, but I'll remove it in my next commit.

Great work here @johnjacobkenny @Mira-Alf. We're on the right track